### PR TITLE
feat(tapable): upgrade DllEntryPlugin - Tapable v1

### PR DESCRIPTION
**Pull request by TheLarkInn**
**What kind of change does this PR introduce?**
Migration of simple plugin (DllEntryPlugin) to use new Tapable v1 API. 

**Did you add tests for your changes?**
N/A

**If relevant, link to documentation update:**
N/A

**Summary**
This upgrades a single plugin to the new plugin hooks usage. I have wanted to get one Plugin upgraded as a POC so that we can then outline how new contributors can help out and upgrade the whole system. 
**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

**Comment by TheLarkInn** [comment]
Agh, @sokra would you mind fixing the diff contribbuter on the commit, I fixed it locally on my computer for future changes.

File: lib/DllEntryPlugin.js [hunk 1]
```diff
@@ -16,16 +16,18 @@ class DllEntryPlugin {
...
-
 			compilation.dependencyFactories.set(SingleEntryDependency, normalModuleFactory);
 		});
-		compiler.plugin("make", (compilation, callback) => {
-			compilation.addEntry(this.context, new DllEntryDependency(this.entries.map((e, idx) => {
+
+		compiler.hooks.make.tapAsync("DllEntryPlugin", (compilation, callback) => {
+			const { context, entries } = this;
+
+			console.log(context, entries);
```
Comment (sokra) [inline lib/DllEntryPlugin.js]: 🤕

Comment (TheLarkInn) [reply lib/DllEntryPlugin.js]: On it! 😅

Comment (TheLarkInn) [reply lib/DllEntryPlugin.js]: We should add an eslint for no consoles <3

**Comment by sokra** [comment]
`git reset next --soft` and commit again. than force push.

**Comment by sokra** [comment]
done by other PR

**Pull request closed without merging**